src/config/dynamic-theme-fixes.config

```diff
@@ -700,6 +700,26 @@ slack.com
 INVERT
 .slack_logo > img
 
+CSS
+@keyframes highlight {
+    20% {
+        background: #595959;
```

Comment:
Please, use `${Light_Mode_Color_Here}` instead of color value to make fix work in light mode too and to apply Filter values properly. Also please add `!important` keyword.